src/index.tsx

```diff
@@ -1,21 +1,26 @@
-import React from 'react'
-import ReactDOM from 'react-dom'
-import { Config as DappProviderConfig, DAppProvider } from '@usedapp/core'
-import { App } from './App'
+import React from "react";
+import { StrictMode } from "react";
+import * as ReactDOMClient from "react-dom/client";
+import { DAppProvider, Config, Goerli } from "@usedapp/core";
 
-const config: DappProviderConfig = {
-  // TODO: Set supported networks
-  // networks: [
-  // Mumbai,
-  // Polygon,
-  // ],
-}
+import App from "./App";
+import { WEB3_PROVIDER_URL } from "./web3";
+
+const config: Config = {
+  readOnlyChainId: Goerli.chainId,
+  readOnlyUrls: {
+    [Goerli.chainId]: WEB3_PROVIDER_URL
+  },
+  networks: [Goerli]
+};
```

Comment:
When we set the supported chain to Goerli, what happens when we switch to Mumbai for contract deployment? Everything goes smoothly?